### ISSUE TITLE
Triggers wrongly name spaced event

### ISSUE THREAD

**Issue by khalwat** [craftcms/digital-products#48]
In `craft\digitalproducts\services\ProductTypes.php` the event `ProductTypeEvent` is used, but it's name spaced as:

`craft\commerce\events\ProductTypeEvent`

...which means that although DigitalProduct's own `ProductTypeEvent` is defined in:

https://github.com/craftcms/digital-products/blob/develop/src/events/ProductTypeEvent.php

...this event is never actually used, and will result in a runtime exception if used as a type, because the Commerce namespace event is the one that DigitalProducts triggers:

https://github.com/craftcms/digital-products/blob/develop/src/services/ProductTypes.php#L257

**Comment by lukeholder** [comment]
Thanks for reporting that! I’ve just fixed it for the next release.

To get the fix early, change your `craftcms/commerce` requirement in composer.json to:

```json
"require": {
  "craftcms/commerce": "dev-develop#50505f00c862216e04c0a09e3f0e6d88625ea3d5 as 2.4.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by lukeholder** [comment]
Fixed in 50505f00c862216e04c0a09e3f0e6d88625ea3d5
